EDSAnalytics/EDSTest/README.md

```diff
@@ -0,0 +1,105 @@
+# EDS Analytics Sample and Test
```

Comment:
I think the `EDSTest` and `EDSTestTests` folders should be renamed to `EDSAnalytics` and `EDSAnalyticsTests` respectively

Reply:
Fixed.